### DISCUSSION TITLE
homebrew/brew repository missing license / copyright notices for bundled gems

### DISCUSSION THREAD

**Discussion by williamfligor** [Homebrew/discussions#3856] [Everyday usage]
It looks like homebrew/brew includes copies of ruby source for many gems, including webrick. Webrick is [licensed](https://github.com/ruby/webrick/blob/master/LICENSE.txt) GPL and unless I misunderstand the requirements, when the lib source is copied to this repo this repo has to also copy and include the notice (LICENSE.txt) in some way. I do not see that in the brew repository for any of the gems.

**Comment by Bo98** [comment]
Webrick should probably not be there at all. I'll sort that and look at the licenses for those that should be there.

The files will likely appear with a `brew install-bundler-gems`, so it's just missing from the repo.
